### ISSUE TITLE
Captcha

### ISSUE THREAD

**Issue by Technetium1** [nortxort/pinylib-rtc#3]
`Unknown command: captcha {u'req': 1, u'tc': u'captcha', u'key': u'XXXXXXXXXXXXXX'}`
Captcha is finally here to cause problems.

**Comment by notnola** [comment]
I have not run into t he need to use captcha, care to explain so maybe I can replicate it and submit a pull request with a fix?

**Comment by ejahglass** [comment]
I actually don't have that error after the first night of the update anymore. Now i'm getting an error whenever anyone cams down the "unpublish" command in pinylib.py will now send a nonetype error. so to temporarily fix it from breaking the script i just deleted the unpublish command, which as far as I could tell was just to notify the console when a user cams down,

**Comment by Technetium1** [comment]
@ejahglass Fix for issue #4 was pushed in https://github.com/nortxort/pinylib-rtc/commit/8312857a6f408fab50c9bd158c43949f685f930f

**Comment by Technetium1** [comment]
Closing until this is an issue again.

**Comment by NawtJ0sh** [comment]
[03:23:04:PM] Unknown command: captcha {u'req': 147, u'tc': u'captcha', u'key': u'6Lfg5x4UAAAAADk1zOokp1zPPFD-WtBvpT4HXa0U'}
[03:23:04:PM] {"key":"6Lfg5x4UAAAAADk1zOokp1zPPFD-WtBvpT4HXa0U","req":147,"tc":"captcha"}
hmm

**Comment by androg55** [comment]
I SECOND THIS ISSUE! Tinychat is requiring a capcha solve

**Comment by ejahglass** [comment]
I'm getting:
Unknown command: captcha {u'req': 1, u'tc': u'captcha', u'key': u'6Lfg5x4UAAAAADk1zOokp1zPPFD-WtBvpT4HXa0U'} 

trying to connect to anything now

**Comment by Nizion** [comment]
Login to the bot account on another tab go into a room do captcha then load not and your good to go best work around atm

**Comment by Nizion** [comment]
Bot *

**Comment by FALCONgod** [comment]
https://github.com/phuein/Tunebot     nort  can you fix this one ???   juat asking   let me know please

https://discord.gg/aQqPmYr

**Comment by ejahglass** [comment]
lmao

**Comment by Nizion** [comment]
FYI you can force trigger the recaptcha by spamming out of a room.

**Comment by jo-nike** [comment]
Loading room in a tab then running bot solution worked well but i moved my bot into cloud on headless server, any idea?!

**Comment by FALCONgod** [comment]
I'm getting he same shit  omg can any one fix this t/c as down today for a short time the did another update omg  nortxort   can  you help in any way   please or  discord me at some time thankyou
Unknown command: captcha {u'req': 1, u'tc': u'captcha', u'key': u'6Lfg5x4UAAAAADk1zOokp1zPPFD-WtBvpT4HXa0U'}

**Comment by FALCONgod** [comment]
can any one over come this captcha  shit

**Comment by ejahglass** [comment]
yeah just log in to the room under bot account, complete captcha and log in
the bot after. sometimes it takes several tries.

**Comment by FALCONgod** [comment]
Yes did that but i need it for beta spam bot captcha shit fix ..not room helper dont use that helper   REGARDS

**Comment by FALCONgod** [comment]
Yes did that but i need it for beta spam bot captcha shit fix ..not room helper dont use that helper   REGARDS

ejahglass 
yeah just log in to the room under bot account, complete captcha and log in
the bot after. sometimes it takes several tries.

did that bro but still no good

**Comment by ejahglass** [comment]
trying to load bot now its saying "Illegal instruction" holy shit, did
tinychat finally figure out how to block us?

**Comment by ejahglass** [comment]
Never mind! ended up actually being an error having to do with overclocking
my raspberry pi, reset config and its all good now.
